src/dataaccess/McodeV09SmartOnFhirDataSource.js

```diff
@@ -68,13 +74,33 @@ class McodeV09SmartOnFhirDataSource extends IDataSource {
             .then(response => this._handleResponseBundle(response.data, prevBundle));
     }
 
+    _handleResponseEntry(entry) {
+        if (entry.resourceType && entry.resourceType === 'Bundle') return entry;
```

Comment:
[DP2] Can we add a comment here about what this function is doing? Looks like it's mapping straight fhir resources into bundle-like objects - is that right?